### DISCUSSION TITLE
Twitch login in self built OBS install?

### DISCUSSION THREAD

**Discussion by Sir-McLightning** [obsproject/obs-studio#11431] [Development Questions] [answered]
I am VERY sorry if this is in the wrong place. I never really come to Github discussions, I just follow OBS for the most part. I searched and found nothing specifically related to this.

I built OBS on Windows from the latest source here. First time ever doing it, process was straight forward in line with that I'm used to. Just one thing though - is the twitch integration not included in that? Cause once it was built and booted, there was no connection option aside from a stream key. I really enjoy testing things on the bleeding edge so I don't mind the bugs when I stream. I'm a bit of a nerd hehe

**Comment by Fenrirthviti** [comment, marked as answer]
Twitch integration uses a custom Twitch app that needs to be registered with Twitch and given a secret key. For obvious reasons, we don't publish that key in the repo for anyone to use. You would need to set up your own version of: https://github.com/obsproject/obs-oauth-cf

The repo contains further documentation.

**Reply by Sir-McLightning** [reply]
Alright! That's fair. Thank you!! Have a wonderful rest of your day 👍
